docs/upgrade-to-v3.md

```diff
@@ -224,6 +224,36 @@ This new version improves the Java API provided to developers. To upgrade your a
 
 > Note: the `DeltaCollectionPage` also offers a `getNextPage` method which simplifies iterating through results and removes the need for consumers to directly handle the next link themselves.
 
+### ChunkedUploadProvider renamed to LargeFileUploadTask
+
+The chunked upload provider has been renamed to larg file upload task and moved from the concurrency package to the tasks package in order to align with SDK design specifications and to clarify the intend of the class. To upgrade your application do the following.
```

Comment:
Love this doc!

`clarify the intend of the class` becomes
`clarify the intent of the class`

Reply:
```suggestion
The chunked upload provider has been renamed to larg file upload task and moved from the concurrency package to the tasks package in order to align with SDK design specifications and to clarify the intent of the class. To upgrade your application do the following.
```